src/styles/index.scss

```diff
@@ -59,6 +59,12 @@
   animation: leafs-animation 18s infinite alternate ease-in-out;
 }
 
+@media (prefers-reduced-motion: reduce) {
+  .leafs-front, .leafs-middle, .leafs-back {
+    animation-name: dissolve;
```

Comment:
I would probably skip do the reverse, and wrap the above animations

Reply:
@nschonni Sounds good, let me go ahead and do that 👍

Reply:
@nschonni I set an animation class that when the `prefers-reduced-motion` is noticed, the animation class sets `animation-name: none`. Let me know if this is what you had in mind or how I could improve upon it. Thank you!